examples/run_example.sh

```diff
@@ -58,8 +58,20 @@ on_exit() {
 
 trap on_exit EXIT
 
+if [[ -z "$EXAMPLE" ]]; then
+    fail "$0 requires an example name as argument"
+    info "usage:\n$0 EXAMPLE"
+    exit 1
```

Comment:
I noticed that the output is currently a bit messy. It looks like this:
```
[FAIL] ./run_example.sh requires an example name as argument
[ usage:
./run_example.sh EXAMPLE ]
[FAIL] Command: "exit 1" exited with exit code: 1
```

I suggest the following changes:
```suggestion
    fail "$0 requires the name of the example as the first argument"
    info "See the README.md file for help"
    exit
```

The script supports a few flags and options now. I think it's easier to document them in the `README.md` file than in the script which is why I opted to remove the usage.

The final result is this:
```
[FAIL] ./run_example.sh requires the name of the example as the first argument
[ See the README.md file for help ]
[FAIL] finished with error
```

Reply:
@p-alik just in case you missed it, this one is still around :)
After that I think we're good to go!